src/main/scala/vct/col/rewrite/PointersToArrays.scala

```diff
@@ -11,7 +11,9 @@ import scala.collection.JavaConverters._
 
 class PointersToArrays(source: ProgramUnit) extends AbstractRewriter(source) {
   def visitType(t: Type): Type = {
-    if(t.isPrimitive(PrimitiveSort.Pointer)) {
+    if (t == null) {
+      t
```

Comment:
I think this may be the same. If you know where a null type is coming from (in a valid expression of course) please let me know